irc.html

```diff
@@ -10,6 +10,9 @@ <h1>Channels</h1>
 <div>
   <p>
     Join one of our <abbr title="Internet Relay Chat">IRC</abbr> channels on Freenode:<br/>
+    Please note that in order to chat on #minetest-hub you need to
+    ask an operator for voice and to chat on #minetest-dev you 
+    need to be using a real IRC client.<br>
```

Comment:
instead of saying "you need a real irc client" just say that you "can't use webchat on -dev"

Reply:
```
  <p>
    Join one of our <abbr title="Internet Relay Chat">IRC</abbr> channels on Freenode:<br/>
    To chat on #minetest-hub you need to ask an operator for voice. You can't you the webchat on #minetest-dev.<br>
  </p>
```
Good?

Reply:
s/can't you/can't use/
yes